features/step_definitions/your_results_steps.rb

```diff
@@ -105,7 +105,10 @@
 
 Then(/^my results should have "([^"]*)" "([^"]*)" and "([^"]*)"$/) do |your_heading, employer_heading, total_heading|
   headings = [your_heading, employer_heading, total_heading]
-  expect(your_results_page.results_period_headings[0..2].map(&:text)).to eq(headings)
+  headings = your_results_page.results_period_headings[0..2].map do |heading|
```

Comment:
I'm trying to work out which `headings` is which. Could you rename the sanitised heading into `sanitised_heading`?

Reply:
I believe @tomas-stefano updated the feature for this PR. I did the front end work and updated the karma test. @tomas-stefano What do you think?

Reply:
I agree about the need to change the name, but I prefer another name. I changed both:

one to actual_headings: the actual headings the app is returning.
the other to expected_headings: the headings the test is expecting to be equal.